### DISCUSSION TITLE
SD Card download and run.

### DISCUSSION THREAD

**Discussion by phil-barrett** [terjeio/ioSender#314] [General]
Is it possible to do a "download and hold"?  If using WiFi, it would be a safety thing to have to press a physical button.  It could be dangerous if a person was on the far side of the moon (or in another room...) and starts the machine. I know people shouldn't do stupid things but sometimes they do...

**Comment by terjeio** [comment]
Add `M0` or `M1` at the start of the file by editing the [postprocessor](https://github.com/grblHAL/PostProcessors), I do that in my VCarve PP. Or compile grblHAL with [this plugin](https://github.com/grblHAL/Templates/tree/master/my_plugin/pause%20on%20SD%20file%20run).
